kqueen/engines/openstack_kubespray.py

```diff
@@ -441,8 +441,14 @@ def _create_group_vars(self):
         src = os.path.join(self.kubespray_path, "inventory/sample/group_vars")
         dst = self._get_cluster_path("group_vars")
         shutil.copytree(src, dst)
+
         with open(os.path.join(dst, "all.yml"), "a") as all_yaml:
-            all_yaml.write("\ncloud_provider: openstack\n")
+            data_to_add = {'cloud_provider': 'openstack'}
```

Comment:
imo unclear naming , maybe kubespray_configuration?

Reply:
kubespray_configuration is located at "all.yml" and we are open this file for append and in data_to_add dictionary contains everything that will be appended